src/constant/constant.js

```diff
@@ -241,3 +241,7 @@ export const WHAT_NEXT = [
         title: 'Step 4',
     },
 ];
+
+export const IMAGES = {
+    DEFAULT_IMAGE: 'https://parkspot.blob.core.windows.net/assets/default.png'
```

Comment:
if we are moving the default in the application level, means this gonna serve as default for all the images used.
Then it would be better to have this in assets rather than getting it from blob store, so that it can be used in any place as a backup image too.